pkg/features/features.go

```diff
@@ -85,6 +85,12 @@ const (
 	// owner: @rrhubenov
 	// alpha: v1.124.0
 	OpenTelemetryCollector featuregate.Feature = "OpenTelemetryCollector"
+
+	// VpaInPlaceOrRecreateUpdateMode enables the usage of in-place Pod resource updates in the Vertical Pod Autoscaler deployments.
+	// resources to perform in-place Pod resource updates.
+	// owner: @vitanovs
+	// alpha: v1.128.0
+	VpaInPlaceOrRecreateUpdateMode featuregate.Feature = "VpaInPlaceOrRecreateUpdateMode"
```

Comment:
Some old VPA related feature gates were `VPAForETCD`, `VPAAndHPAForAPIServer`. Hence, I vote the naming of this feature to follow the same naming pattern - i.e use `VPA<...>` instead of `Vpa<...>`.

```suggestion
	VPAInPlaceOrRecreateUpdateMode featuregate.Feature = "VPAInPlaceOrRecreateUpdateMode"
```

Reply:
Thanks for the suggestion, updated the _feature gate_ to `VPAInPlaceUpdates` in my latest commits.